test/bats/helpers/lxly-bridge.bash

```diff
@@ -89,6 +89,55 @@ function bridge_asset() {
     fi
 }
 
+function get_bridges() {
+    local aggkit_node_url=$1
+    local l1_rpc_network_id=$2
+
+    bridges_result=$(cast rpc --rpc-url "$aggkit_node_url" "bridge_getBridges" "$l1_rpc_network_id")
+
+    echo "------ bridges_result ------"
+    echo "$bridges_result"
+    echo "------ bridges_result ------"
+
+    count=$(jq -r '.count' <<< "$bridges_result")
+    if [ "$count" -ne 1 ]; then
+        echo "Assertion failed. 'count' should be 1. Actual: $count"
+        exit 1
+    fi
+
+    required_fields=(
+        "block_num"
+        "block_pos"
+        "block_timestamp"
+        "leaf_type"
+        "origin_network"
+        "origin_address"
+        "destination_network"
+        "destination_address"
+        "amount"
+        "deposit_count"
+        "tx_hash"
+        "from_address"
+        "bridge_hash"
+        "metadata"
+    )
+
+    # Check that all required fields exist (and are not null) in bridges[0]
+    for field in "${required_fields[@]}"; do
+        value=$(jq -r --arg fld "$field" '.bridges[0][$fld]' <<< "$bridges_result")
+        if [ "$value" = "null" ] || [ -z "$value" ]; then
+            echo "Assertion failed. Missing or null '$field' in the first bridge object."
+            exit 1
+        fi
+    done
+
+    origin_network=$(jq -r '.bridges[0].origin_network' <<< "$bridges_result")
+    if [ "$origin_network" -ne 0 ]; then
+        echo "Assertion failed. 'origin_network' should be 0. Actual: $origin_network"
+        exit 1
+    fi
```

Comment:
This should be also asserted on the outside (in the particular test). This helper function, IMO should just return a `bridge` response, based on a provided `tx_hash` (or error in case it is not found).